pkg/client/v1/nimbleos/volume.go

```diff
@@ -268,7 +268,7 @@ type Volume struct {
 	// PreFilter - Pre-filtering criteria.
 	PreFilter *string `json:"pre_filter,omitempty"`
 	// AvgStatsLast5mins - Average statistics in last 5 minutes.
-	AvgStatsLast5mins *NsAverageStats `json:"avg_stats_last_5mins,omitempty"`
+	AvgStatsLast5mins *NsAverageStats `json:"avg_stats_last5mins,omitempty"`
```

Comment:
Oh, ouch.  This defect keeps recurring.  It should have caused a test failure this time, at least.

Reply:
What is the resolution here?  It looks like the missing underscore defect is still present.  A fix is needed in the generator.  Until that happens, a manual fix in the generated code will be required after each run.  (Probably this was fixed before another generator run was done for the otp file name change?  Manual changes are a pain.)

Reply:
Sorry I missed this. Working on a fix right now!